src/components/Layout.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+
+import { Navbar } from './header/Navbar';
+import { Footer } from './footer/Footer';
+
+export const Layout = ({ children }) => {
+  return (
+    <>
+      <Navbar />
+      {children}
```

Comment:
children można przekazać w tagu main, mimo że to react to nadal obowiązują zasady semantycznego htmla